src/js/view/controls/tizen/tizen-controls.ts

```diff
@@ -185,9 +249,12 @@ class TizenControls extends Controls {
     }
 
     disable(model: ViewModel): void {
+        this.api = null;
+
         if (this.keydownCallback) {
             document.removeEventListener('keydown', this.keydownCallback);
         }
+
```

Comment:
This listener should be removed by context (add `this` as a third argument and then use `api.off(null, null, this)` to remove the listener.
```js
api.on('playlistComplete', () => {
    this.onBackClick(api);
});
```

```js
api.on('playlistComplete', () => {
    this.onBackClick(api);
}, this);
```

```suggestion
        api.off(null, null, this)'
```